draft-ietf-lamps-kyber-certificates.md

```diff
@@ -309,10 +321,71 @@ OneAsymmetricKey is replicated below.
   2021 ASN.1 syntax {{X680}}.
 </aside>
 
-When used in a OneAsymmetricKey type, the privateKey OCTET STRING contains
-the raw octet string encoding of the 64-octet seed. The publicKey field
-SHOULD be omitted because the public key can be computed as noted earlier
-in this section.
+For ML-KEM private keys, the privateKey field in OneAsymmetricKey contains
+one of the following CHOICE structures encoded as an OCTET STRING. The seed
+format is a fixed 64 bytes for all security levels, while the expandedKey
+and both formats vary in size by security level:
+
+~~~
+  ML-KEM-512-PrivateKey ::= CHOICE {
+    seed [0] OCTET STRING (SIZE (64)),
+    expandedKey OCTET STRING (SIZE (1632)),
+    both SEQUENCE {
+      seed OCTET STRING (SIZE (64)),
+      expandedKey OCTET STRING (SIZE (1632))
+      }
+    }
+
+  ML-KEM-768-PrivateKey ::= CHOICE {
+    seed [0] OCTET STRING (SIZE (64)),
+    expandedKey OCTET STRING (SIZE (2400)),
+    both SEQUENCE {
+      seed OCTET STRING (SIZE (64)),
+      expandedKey OCTET STRING (SIZE (2400))
+      }
+    }
+
+  ML-KEM-1024-PrivateKey ::= CHOICE {
+    seed [0] OCTET STRING (SIZE (64)),
+    expandedKey OCTET STRING (SIZE (3168)),
+    both SEQUENCE {
+      seed OCTET STRING (SIZE (64)),
+      expandedKey OCTET STRING (SIZE (3168))
+      }
+    }
+
+~~~
+
+The CHOICE allows three representations of the private key:
+
+* The seed format (tag [0]) contains just the 64-byte seed value
+from which both the expanded private key and public key can be
+derived using `ML-KEM.KeyGen_internal(d,z)` (algorithm 16) using
+the first 32 octets as *d* and the remaining 32 octets as *z*.
+
+* The expandedKey format contains the full expanded private key
```

Comment:
Nit: I'd leave out the full, as the key could be expanded even more. (Matrix A.)